File: Aplikacija/Back deo/ERestoran/Models/Sala.cs
namespace Models
{
    public class Sala
    {
        [Key]
        public int ID { get; set; }
        public required double Duzina { get; set; } //u metrima
        public required double Sirina { get; set; } //u metrima
        public List<Sto>? Stolovi { get; set; }
        public List<ObjekatSale>? Predmeti { get; set; }
        [JsonIgnore]
        public Restoran? Restoran { get; set; }
    }
}

File: Aplikacija/Front deo/erestoranFront/src/app/feature/restaurants/models/Sala.ts
import { ObjekatSale } from "./ObjekatSale";
import { Sto } from "./Sto";

export interface Sala{
    id: number;
    sirina: number;
    duzina: number;
    stolovi: Sto[];
    predmeti: ObjekatSale[];
}